src/firecfg/firecfg.config

```diff
@@ -895,6 +895,7 @@ vivaldi-beta
 vivaldi-snapshot
 vivaldi-stable
 vlc
+vmplayer
```

Comment:
If VMWare usage under Firejail shows the `kernel modules` problem like #5861 suggests, IMO it would make more sense to `disable` the related commands with a note. Suppose a package manager hook for firejail runs firecfg post-install. Some time later a user installs VMWare. When auto-sandboxed, its kernel modules never actually get installed. GUI-minded users won't have much indication of what's going on...

Reply:
Done.